internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -0,0 +1,122 @@
+package vm_console_proxy
+
+import (
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/internal/operands"
+	vm_console_proxy_bundle "kubevirt.io/ssp-operator/internal/vm-console-proxy-bundle"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	operandName      = "vm-console-proxy"
+	operandComponent = ""
```

Comment:
@ksimon1 @akrejcir Can you please suggest what should be the value of `operandComponent`?

Reply:
The value is used as the recommended label `app.kubernetes.io/component` : https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

I'm not sure where is this label used, so I think the value can simply be `console-proxy`, or `vm-console-proxy`.